layer.md

```diff
@@ -186,6 +186,38 @@ rootfs-c9d-v1.s1/
         my-app-tools
 ```
 
+### Symbolic Links
+
+When applying a changeset, (assessing the changeset, or creating the original comparison filesystem) symlinks that are either:
+
+- Already existing in a previous layer
+- Added in the current layer
+
+SHOULD NOT be followed. 
+If the path of a changeset contains a symlink component, the symlink component MUST be removed and replaced with a directory as well as any necessary subdirectories to reconstruct the path of the changeset. 
```

Comment:
with the above change, then this will stand more on its own, which makes it more readable IMO